requirements.txt

```diff
@@ -5,3 +5,20 @@
 Adafruit-Blinka
 adafruit-circuitpython-register
 adafruit-circuitpython-busdevice
+
+PyVISA~=1.13.0
+numpy~=1.26.0
+requests~=2.31.0
+matplotlib~=3.9.1
+pyserial~=3.5
+pandas~=2.1.1
+colorama~=0.4.6
+nidaqmx~=1.0.0
+VotschTechnikClimateChamber~=0
```

Comment:
Why have all of these modules been added to the requirements.txt file?

Are any / all of these absolutely necessary? Many of them don't have analogous modules in CircuitPython so there functionality will not be possible to use on microcontrollers.

I am in favor of removing these, unless if there are any that are strictly necessary for the operation of this library then they can remain.